lightning/src/ln/channelmanager.rs

```diff
@@ -248,7 +248,7 @@ pub(super) enum HTLCForwardInfo {
 /// variable at runtime (eg Channel::channel_reestablish needs to re-send messages in the order
 /// they were originally sent). In those cases, this enum is also returned.
 #[derive(Clone, PartialEq)]
-pub(super) enum RAACommitmentOrder {
+pub enum RAACommitmentOrder {
```

Comment:
Hmm, I'm curious why you need this one - its definitely intended as a super internal type, and is mostly used to order the events that get pushed into message the pending_msg_events Vec, which generally should be accessed via <ChannelManager as MessageSendEventsProvider>::get_and_clear_pending_msg_events().

Reply:
Ah, I don't need it, will remove.